swarmform/core/__init__.py

```diff
@@ -0,0 +1,15 @@
+# These imports allow a much simpler import of core Fireworks functionality.
+# E.g., you can now do "from fireworks import Firework", instead of from
+# "fireworks.core.firework import Firework".
+from fireworks.core.firework import FiretaskBase, FireTaskBase, Firework, Launch, Workflow, \
+    FWAction, Tracker
+from fireworks.core.fworker import FWorker
+from fireworks.core.launchpad import LaunchPad
+from fireworks.utilities.fw_utilities import explicit_serialize
+from fireworks.user_objects.firetasks.script_task import ScriptTask, PyTask
```

Comment:
Are we using these imports? Looks like we are not using many of them

Reply:
Fixed